tools/debian_buildenv_qt6.sh

```diff
@@ -100,10 +100,6 @@ case "$1" in
             protobuf-compiler \
             qtkeychain-qt6-dev \
             qt6-declarative-dev \
-            qml-module-qtquick-controls \
-            qml-module-qtquick-controls2 \
-            qml-module-qt-labs-qmlmodels \
-            qml-module-qtquick-shapes \
```

Comment:
I thought this is just about removing them from Qt5.
Why are these removed from the Qt6 buildenv.
Won't cmake miss them when building with QT6=ON?

Sorry, I didn't follow the Qt6 topic lately, did I miss the fact that QML was removed _entirely_ from 2.4?

Reply:
These are the Qt5 variants. We may need the Qt6 variants, like in https://github.com/mixxxdj/mixxx/pull/12535 which is targeted to 2.5. Should we use the 2.5 file here as well? I original considered as unnecessary because we have also artificially failed the Qt6 build in this branch when not on GitHub. What do you think?

Reply:
I don't think so.
If the CI Qt6 build passes we should not complicate things in order to allow a QML build of Mixxx 2.4, that's what 2.5 is for.